packages/pn-pa-webapp/src/components/NewNotification/__test__/PreliminaryInformationsContent.test.tsx

```diff
@@ -0,0 +1,131 @@
+import { Formik, FormikProps } from 'formik';
+import { vi } from 'vitest';
+
+import {
+  fireEvent,
+  getById,
+  render,
+  screen,
+  testFormElements,
+} from '@pagopa-pn/pn-commons/src/test-utils';
+
+import { NewNotificationLangOther } from '../../../models/NewNotification';
+import { PreliminaryInformationsPayload } from '../../../redux/newNotification/types';
+import PreliminaryInformationsContent from '../PreliminaryInformationsContent';
+
+// mock imports
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (str: string) => str,
+    i18n: { language: 'it' },
+  }),
+}));
+
+describe('PreliminaryInformationsContent', () => {
+  const mockFormikProps = {
+    values: {
+      lang: 'it',
+      subject: '',
+      abstract: '',
+      additionalLang: '',
+      additionalSubject: '',
+      additionalAbstract: '',
+    },
+    touched: {},
+    errors: {},
+    handleChange: jest.fn(),
+  } as unknown as FormikProps<PreliminaryInformationsPayload>;
```

Comment:
We mock data in rare case, because mocking can lead to a cases that are different from the real ones. So instead of mocking, create a test component here and in it init formik and pass it to the PreliminaryInformationsContent

Reply:
Ok, I'll keep it in mind for the next tests